python/lsst/ts/m2/csc.py

```diff
@@ -81,6 +82,10 @@ class M2(salobj.ConfigurableCsc):
     # Command timeout in second
     COMMAND_TIMEOUT = 10
 
+    # Limits of force in Newton
+    LIMIT_FORCE_AXIAL = 444.82  # 100 lbf
```

Comment:
A couple things here:

* where are these numbers derived from? is there a document or something you could reference here
* Consider moving these to a `constants.py` submodule.
* Consider renaming them such that "what this is" comes first, e.g. `FORCE_LIMIT_AXIAL`, `FORCE_LIMIT_TANGENT`.

Reply:
Updated in the commit: [a4aa0a8](https://github.com/lsst-ts/ts_m2/pull/35/commits/a4aa0a884c2f02ed4866ffd4a78151e16563f995). For the `constants.py`, I will update and reuse them at the next time when I update the `ts_m2com`. There are the duplicated values in that module.

Thanks!